superset/migrations/versions/2024-08-13_15-29_a6b32d2d07b1_remove_sl_columns.py

```diff
@@ -34,9 +34,14 @@
 
 def upgrade():
     connection = op.get_bind()
-    if connection.dialect.name != "sqlite":
-        drop_fks_for_table("sl_columns")
-    op.drop_table("sl_columns")
+
+    try:
+        if connection.dialect.name != "sqlite":
```

Comment:
also seems like the special handling for sqlite could be handled within `drop_fks_for_table`. More generally would be great to have migrations that are simpler / self-healing / free-of-engine-specific-logic through better utils.